addons/stage-monitor/addon.json

```
@@ -0,0 +1,152 @@
+{
+  "name": "Customizable stage monitors",
+  "description": "Changes the color and roundness of variable and list monitors on the stage.",
+  "credits": [
+    {
+      "name": "Samq64",
+      "link": "https://github.com/Samq64"
+    },
+    {
+      "name": "Maximouse",
+      "link": "https://scratch.mit.edu/users/Maximouse/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "monitors.css",
+      "matches": ["projects"]
+    },
+    {
+      "url": "monitor-values.css",
+      "matches": ["projects"],
+      "if": {
+        "settings": { "customValueColor": true }
+      }
+    }
+  ],
+  "customCssVariables": [
+    {
+      "name": "monitorLabel",
+      "value": {
+        "type": "textColor",
+        "source": {
+          "type": "settingValue",
+          "settingId": "monitor"
+        }
+      }
+    },
+    {
+      "name": "monitorValueText",
+      "value": {
+        "type": "textColor",
+        "source": {
+          "type": "settingValue",
+          "settingId": "monitorValueBg"
+        }
+      }
+    },
+    {
+      "name": "listHeaderText",
+      "value": {
+        "type": "textColor",
+        "source": {
+          "type": "settingValue",
+          "settingId": "listHeader"
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "name": "Monitor color",
+      "id": "monitor",
+      "type": "color",
+      "allowTransparency": true,
+      "default": "#e5f0ff"
+    },
+    {
+      "name": "List header color",
+      "id": "listHeader",
+      "type": "color",
+      "allowTransparency": true,
+      "default": "#ffffff"
+    },
+    {
+      "name": "Monitor roundness (px)",
+      "id": "monitorRadius",
+      "type": "integer",
+      "min": 0,
+      "max": 11,
+      "default": 4
+    },
+    {
+      "name": "Value roundness (px)",
+      "id": "valueRadius",
+      "type": "integer",
+      "min": 0,
+      "max": 11,
+      "default": 4
+    },
+    {
+      "name": "Custom value color",
+      "id": "customValueColor",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Value color",
+      "id": "monitorValueBg",
+      "type": "color",
+      "allowTransparency": true,
+      "default": "#333333",
```

```suggestion
      "default": "#575e75",
```
The default should be a color from Scrath's color palette.